multi-node/libvirt/kube-worker/openstack/latest/user_data

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash
```

Comment:
If the only difference here from multi-node/generic is the etcd/controller env vars, maybe instead we override them in a separate ENV_FILE (see line 20 of this file).